openwisp_ipam/api/views.py

```diff
@@ -164,6 +164,7 @@ class SubnetListCreateView(ListCreateAPIView):
     serializer_class = SubnetSerializer
     authentication_classes = (BearerAuthentication, SessionAuthentication)
     permission_classes = (DjangoModelPermissions,)
+    throttle_scope = 'ipam'
```

Comment:
Why only this view?
What about the others?
Notice here it is added in a Mixin, so that all views using the mixin get it: https://github.com/openwisp/openwisp-firmware-upgrader/pull/75/files

Reply:
Yes @atb00ker, I will do it in this manner, and push it ASAP.

Reply:
Done @atb00ker, Please have a look.